### PR TITLE
Fix and improve components

### PR DESCRIPTION
**Pull request by HichamELBSI**
Fix all feedback on the in progress column : https://www.notion.so/strapi/f15d5f0f7cfb4c34ab802408c0f41ea7?v=a611848256ca4488ab4d9a02a03de5fe

### CODE REVIEW

**Review by soupette — APPROVED** [review]
LGTM

**Review by ronronscelestes — APPROVED** [review]
LGTM 🔥

File: packages/strapi-parts/src/Text/index.js [hunk 1]
```diff
@@ -49,14 +49,21 @@ export const Subtitle = styled(Text)`
...
 
-export const TableLabel = styled(TextButton)`
+export const TableLabel = styled(ButtonText)`
   font-weight: 600;
   font-size: ${11 / 16}rem;
   line-height: 1.45;
   text-transform: uppercase;
 `;
+
+export const EllipsisText = styled(Text)`
```
Comment (bulby97) [inline packages/strapi-parts/src/Text/index.js:64]: WDYT about having ellipsis as a component and not a prop ?
Some might use a TableLabel, ButtonText, or any other Text component with an ellipsis prop on it

**Review by bulby97 — CHANGES_REQUESTED** [review]
Good job ;)

File: packages/strapi-parts/src/Text/index.js (hunk 1 above)
Comment (HichamELBSI) [reply packages/strapi-parts/src/Text/index.js:64]: I understand that but the problem is that the Text component is a span. To use the ellipsis style, we need a `block` element not an `inline`. I'm afraid that moving the text component to a `block` will break a lot of things in the other components, layouts or migrated views. @bulby97

Comment (bulby97) [reply packages/strapi-parts/src/Text/index.js:64]: Got your point, but ellipsis can also be applied to a inline-block. I don't think the inline block can break inline stuff, but maybe I'm wrong

Comment (HichamELBSI) [reply packages/strapi-parts/src/Text/index.js:64]: You are right, I can add the ellipsis to the inline-block my bad.

**Pull request merged**